### PR TITLE
Correct documentation on Candlepin certs

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ for communication between services inside the Katello deployment.
 | ${hostname}-parent-cert       | a client certificate to read content from Pulp parent (distributed to the child over qpid) | default |
 | ${hostname}-qpid-broker       | a client certificate for qpid broker                                                       | default |
 | ${hostname}-qpid-client-cert  | a client certificate for Pulp to connect to qpid                                           | default |
-| java-client                   | a client certificate for Candlpin to connect to qpid                                       | default |
+| ${hostname}-tomcat            | a server certificate for Candlepin http                                                    | default |
```
Comment (ekohl) [inline README.md]: This may not be right. It looks like this is used as intermediates to import into the NSS db.

Comment (ehelms) [reply README.md]: Without doing a hard check, I'd think this certificate was for building up the trustore and keystore for use by Tomcat server. IIRC nssdb is only used for qpidd.

File: README.md [hunk 2]
```diff
@@ -38,7 +38,8 @@ for communication between services inside the Katello deployment.
...
+| java-client                   | a client certificate for Candlepin to connect to qpid                                      | default |
```
Comment (ekohl) [inline README.md]: @ehelms I do think this line is correct. If so, I'll revert the other one.

Comment (ehelms) [reply README.md]: Agreed.

**Comment by ekohl** [comment]
Since now it was just a trivial typo fix I pushed it to master.

**Pull request merged**